src/Cartography.StaticMap.Abstractions/IStaticMapComponent.cs

```diff
@@ -1,6 +1,6 @@
-﻿using Windows.Foundation;
+﻿using System.Drawing;
```

Comment:
Hmmm do you really intend to use System.Drawing APIs?

Reply:
What is it used for?

Reply:
To store map Size width / length